README.md

```diff
@@ -96,7 +96,7 @@ namespace DotNetCore_Template.Controllers
 }
 ```
 
-Visiting the defined route: **/hello** through a **GET** request, you will receive the `Hello World!` message as response. 
+Visiting the defined route: **/hello** through a **GET** request, you will receive the `Hello World!` message as response.
```

Comment:
This line is useless, there should be no `/hello` handler

Reply:
Actually this part of the readme explains how to add an hello handler to the template! Therefore, I think that this line is correct!

Reply:
ok fine